src/util/object_factory_parameters.h

```diff
@@ -14,6 +14,9 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <list>
 
 #include <util/irep.h>
+#include <util/optional.h>
+#include <analyses/interval_domain.h>
```

Comment:
I think we shouldn't have dependencies of util on analyses, so the best solution is probably to move `interval_template.h` to  `util` (and we can drop "template" from the name)

Reply:
Yep, moved... with a bunch of other files it needed...